packages/editor-ui/src/components/CommunityPackageManageConfirmModal.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<Modal
+		width="540px"
+		:name="COMMUNITY_PACKAGE_CONFIRM_MODAL_KEY"
+		:title="getModalContent.title"
+		:eventBus="modalBus"
+		:center="true"
+		:showClose="!isLoading"
+		:beforeClose="onModalClose"
+	>
+		<template slot="content">
+			<n8n-text>{{ getModalContent.message }}</n8n-text>
+			<div :class="$style.descriptionContainer" v-if="this.mode === COMMUNITY_PACKAGE_MANAGE_ACTIONS.UPDATE">
+				<div :class="$style.descriptionIcon">
+					<n8n-icon icon="info-circle" size="xlarge" />
+				</div>
+				<div :class="$style.descriptionText">
+					<n8n-text>{{ getModalContent.description }}</n8n-text>
+				</div>
+			</div>
+		</template>
+		<template slot="footer">
+			<n8n-button
+				:loading="isLoading"
+				:disabled="isLoading"
+				:label="isLoading ? getModalContent.buttonLoadingLabel : getModalContent.buttonLabel"
+				size="large"
+				float="right"
+				@click="onConfirmButtonClick"
+			/>
+		</template>
+	</Modal>
+</template>
+
+<script>
+import Vue from 'vue';
+import mixins from 'vue-typed-mixins';
+import Modal from './Modal.vue';
+import { COMMUNITY_PACKAGE_CONFIRM_MODAL_KEY, COMMUNITY_PACKAGE_MANAGE_ACTIONS } from '../constants';
+import { mapGetters } from 'vuex';
+import { showMessage } from './mixins/showMessage';
+
+export default mixins(showMessage).extend({
+	name: 'CommunityPackageManageConfirmModal',
+	components: {
+		Modal,
+	},
+	props: {
+		modalName: {
+			type: String,
+			required: true,
+		},
+		activePackageName: {
+			type: String,
+			required: true,
+		},
+		mode: {
+			type: String,
+		},
+	},
+	data() {
+		return {
+			modalBus: new Vue(),
+			COMMUNITY_PACKAGE_CONFIRM_MODAL_KEY,
+			COMMUNITY_PACKAGE_MANAGE_ACTIONS,
+		};
+	},
+	computed: {
+		...mapGetters('communityNodes', ['isLoading']),
```

Comment:
Since they are not used in parallel, no. The only problem is that package list will show loading state while something is installing/updating/uninstalling from the modal which is not necessarily incorrect state for it, so I left it so like this.
Can separate those states if needed.